### PR TITLE
docs: fix site url not found

### PR DESCRIPTION
**Pull request by googs1025**
- fix site url

### CODE REVIEW

File: site/content/en/docs/contribution guidelines/_index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ description: >
   How to contribute to Jobset
 ---
 
-Welcome to Kubernetes. We are excited about the prospect of you joining our [community](https://git.k8s.io/community)! The Kubernetes community abides by the CNCF [code of conduct](code-of-conduct.md). Here is an excerpt:
+Welcome to Kubernetes. We are excited about the prospect of you joining our [community](https://git.k8s.io/community)! The Kubernetes community abides by the CNCF [code of conduct](../code-of-conduct.md). Here is an excerpt:
```
Comment (danielvegamyhre) [inline site/content/en/docs/contribution guidelines/_index.md:9]: The code of conduct link still doesn't work in the preview

**Review by danielvegamyhre — COMMENTED** [review]
Thanks for fixing this, left one comment

File: site/content/en/docs/contribution guidelines/_index.md (hunk 1 above)
Comment (googs1025) [reply site/content/en/docs/contribution guidelines/_index.md]: thank for catching that. fix!

Comment (danielvegamyhre) [reply site/content/en/docs/contribution guidelines/_index.md]: Let's just have a copy of the code-of-conduct.md in the site/ directory somewhere so we don't need to use this super long relative path

Comment (googs1025) [reply site/content/en/docs/contribution guidelines/_index.md]: I referred to the site/content/en/docs/reference directory and also placed the code-of-conduct.md in the site/content/en/docs/contribution guidelines directory

Comment (googs1025) [reply site/content/en/docs/contribution guidelines/_index.md]: https://github.com/googs1025/jobset/blob/jobset_sit/site/content/en/docs/contribution%20guidelines/_index.md
This is a bit strange. It works on my branch.

Comment (danielvegamyhre) [reply site/content/en/docs/contribution guidelines/_index.md]: Maybe the preview link here is outdated? https://github.com/kubernetes-sigs/jobset/pull/541#issuecomment-2067821199 Not sure, it is still live though so seems like it should reflect the latest changes

File: site/content/en/docs/contribution guidelines/_index.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ description: >
   How to contribute to Jobset
 ---
 
-Welcome to Kubernetes. We are excited about the prospect of you joining our [community](https://git.k8s.io/community)! The Kubernetes community abides by the CNCF [code of conduct](code-of-conduct.md). Here is an excerpt:
+Welcome to Kubernetes. We are excited about the prospect of you joining our [community](https://git.k8s.io/community)! The Kubernetes community abides by the CNCF [code of conduct](./code-of-conduct.md). Here is an excerpt:
```
Comment (kannon92) [inline site/content/en/docs/contribution guidelines/_index.md:9]:
```suggestion
Welcome to Kubernetes. We are excited about the prospect of you joining our [community](https://git.k8s.io/community)! The Kubernetes community abides by the CNCF [code of conduct](https://git.k8s.io/community/code-of-conduct.md). Here is an excerpt:
```

File: site/content/en/docs/contribution guidelines/code-of-conduct.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Kubernetes Community Code of Conduct
```
Comment (kannon92) [inline site/content/en/docs/contribution guidelines/code-of-conduct.md:1]: Let's just drop this file and use the link for the kubernetes community (Kueue does this also).

Comment (googs1025) [reply site/content/en/docs/contribution guidelines/code-of-conduct.md]: done!  That makes sense, thanks for the suggestion from @kannon92

**Comment by danielvegamyhre** [comment]
/lgtm
/approve

Thanks!

**Pull request merged**